docs/sts/sts.md

```diff
@@ -0,0 +1,73 @@
+# Using manual mode with STS
+
+In a STS Cluster, *CredentialsRequests* are not automatically provisioned by
+the `cloud-credential-operator` and requires manual intervention by the
+cluster-admin. They need to be provisioned using the `ccoctl` binary.
+
+The `aws-load-balancer-operator` also relies on the `cloud-credential-operator`
+to provision the *CredentialsRequest* so in an STS Cluster, it needs to be
+provisioned manually. The operator will wait until the required secrets are
+created and available.
+
+## Pre-Requisites
+
+### Extract and prepare the `ccoctl` binary
+
+1. Obtain the OpenShift Container Platform release image:
+
+    ```bash
+    RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
+    ```
+
+2. Get the CCO container image from the OpenShift Container Platform release image:
+
+    ```bash
+    CCO_IMAGE=$(oc adm release info --image-for='cloud-credential-operator' $RELEASE_IMAGE)
+    ```
+
+3. Extract the ccoctl binary from the CCO container image within the OpenShift
+Container Platform release image:
+
+    ```bash
+    oc image extract $CCO_IMAGE --file="/usr/bin/ccoctl" -a ~/.pull-secret
+    ```
+
+4. Change the permissions to make ccoctl executable:
+
+    ```bash
+    chmod 775 ccoctl
+    ```
+
```

Comment:
Let's just link to the existing documentation where they describe how to get _ccoctl_.